doc/GAME_BALANCE.md

```diff
@@ -124,6 +124,22 @@ Very high difficulty: ~100 (supernaturally dangerous; zombie hulk, shoggoth, wra
 Maximum danger: 300 (highest for balance purposes; triffid heart, melded task force)
 
 
+# Monster Vision scaling:
+Minimum: 0 (blind; yellow chick, fly, goose)
```

Comment:
This is gonna need some clarification that those mentioned here are night visions. I was confused when I read that and got the message that geese are blind

Reply:
Yeah, I might split them into night and day just for clarity

Reply:
Should be clear now